concordium-std-derive/src/lib.rs

```diff
@@ -2417,17 +2417,17 @@ fn impl_state_clone(ast: &syn::DeriveInput) -> syn::Result<TokenStream> {
                             field_names.extend(quote!(#field_ident,));
                         }
                         let pattern = quote!((#field_names));
-                        (quote!(#data_name::#variant_ident#pattern), pattern)
+                        (quote!(#data_name::#variant_ident #pattern), pattern)
                     }
                     syn::Fields::Unit => (
-                        quote!(#data_name::#variant_ident#field_names),
+                        quote!(#data_name::#variant_ident #field_names),
                         proc_macro2::TokenStream::new(),
                     ),
                 };
                 let variant_ident = &variant.ident;
 
                 matches_tokens.extend(quote! {
-                    #data_name::#variant_ident#pattern => {
+                    #data_name::#variant_ident #pattern => {
```

Comment:
Where do these changes come from?

Reply:
`cargo fix --edition`. It is related to the new reserved syntax mentioned above.